HISTORY.md

```diff
@@ -1,3 +1,106 @@
+## v2.5.0b1 (2023-11-09)
+
+[GitHub release](https://github.com/pydantic/pydantic/releases/tag/v2.5.0b1)
+
+### What's Changed
+
+#### Packaging
+
+* Update `pydantic-core` to 2.11.0 by @davidhewitt in [#7852](https://github.com/pydantic/pydantic/pull/7852)
+* Bump `pydantic-core` to `2.12.0` by @sydney-runkle in [#8029](https://github.com/pydantic/pydantic/pull/8029)
+* uprev pydantic-core to v2.13.0 by @samuelcolvin in [#8045](https://github.com/pydantic/pydantic/pull/8045)
+
+#### New Features
+
+* Allow plugins to catch non ValidationError errors by @adriangb in [#7806](https://github.com/pydantic/pydantic/pull/7806)
+* Support __doc__ argument in create_model() by @chris-spann in [#7863](https://github.com/pydantic/pydantic/pull/7863)
+* Expose regex_engine flag by @utkini in [#7768](https://github.com/pydantic/pydantic/pull/7768)
```

Comment:
```suggestion
* Expose `regex_engine` flag by @utkini in [#7768](https://github.com/pydantic/pydantic/pull/7768)
```
need to extend this line to explain you can now choose between the rust or python regex engines.